src/game-servers/providers/static-game-server/controllers/static-game-servers.controller.ts

```diff
@@ -22,19 +21,22 @@ import { Secret } from '@/auth/decorators/secret.decorator';
 import { GameServerHeartbeat } from '../dto/game-server-heartbeat';
 import { RealIp } from 'nestjs-real-ip';
 import { SecretPurpose } from '@/auth/secret-purpose';
+import { StaticGameServersService } from '../services/static-game-servers.service';
+import { GameServersService } from '@/game-servers/services/game-servers.service';
 
-@Controller('game-servers')
-export class GameServersController {
+@Controller(['game-servers/static', 'game-servers'])
```

Comment:
🤯 woah

Reply:
Yeah, I'll make some kind of proxy controller that marks all of `game-servers` subroutes deprecated.